modules/effects/arrow-effects-data/src/main/kotlin/arrow/effects/typeclasses/MonadDeferCancellableContinuations.kt

```diff
@@ -17,13 +19,16 @@ typealias Disposable = () -> Unit
 
 @RestrictsSuspension
 @Suppress("DELEGATED_MEMBER_HIDES_SUPERTYPE_OVERRIDE")
-open class MonadDeferCancellableContinuation<F, A>(SC: MonadDefer<F>, override val context: CoroutineContext = EmptyCoroutineContext) :
-  MonadErrorContinuation<F, A>(SC), MonadDefer<F> by SC {
+open class MonadDeferCancellableContinuation<F, A>(val SC: MonadDefer<F>, override val context: CoroutineContext = EmptyCoroutineContext) :
```

Comment:
Doesn't need to be a field as it's available as `this` via delegation.

Reply:
Already addressed